const handleAxiosError = (errorMessage) => {
    console.log("Ohh, we follow error: " + errorMessage)
}

export default handleAxiosError